src/components/datagrid/body/data_grid_cell_actions.tsx

```diff
@@ -69,7 +67,6 @@ export const EuiDataGridCellActions = ({
         iconSize="s"
       />
     );
-
     const [visibleCellActions] = getVisibleCellActions(
```

Comment:
accidental merge shenanigans

```suggestion
    );

    const [visibleCellActions] = getVisibleCellActions(
```